src/main/kotlin/step2/Operator.kt

```diff
@@ -0,0 +1,18 @@
+package step2
+
+enum class Operator(val operator: String, val operation: (Int, Int) -> Int) {
+    PLUS("+", operation = { x, y -> x + y }),
+    MINUS("-", operation = { x, y -> x - y }),
+    MULTIPLY("*", operation = { x, y -> x * y }),
+    DIVIDE("/", operation = { x, y -> x / y });
```

Comment:
[Named Arguments](https://kotlinlang.org/docs/functions.html#default-arguments)와  Kotlin 스러운 함수형 인터페이스([SAM](https://kotlinlang.org/docs/fun-interfaces.html))활용 👍💯

추가적으로 operator 프로퍼티는 Named Arguments로 선언하지 않았고, operation의 경우에는 Named Arguments로 선언하셨네요.   이 부분을 통일해보면 어떨까요? 사용한다면 다 같이 사용하고, 사용하지 않는다면 다 같이 사용하지 않는 방식으로요! 😃

Reply:
이름있는 아규먼트을 사용한다면 모든 아규먼트에 사용하는게 좋겠네요!

```suggestion
    PLUS(symbol = "+", operation = { x, y -> x + y }),
    MINUS(symbol = "-", operation = { x, y -> x - y }),
    MULTIPLY(symbol = "*", operation = { x, y -> x * y }),
    DIVIDE(symbol = "/", operation = { x, y -> x / y });
```